distribution/lib/Standard/Base/0.0.0-dev/src/Data/Time/Date_Time.enso

```diff
@@ -584,18 +585,8 @@ type Date_Time
                 Time_Utils.datetime_adjust self Time_Utils.AdjustOp.MINUS period.internal_period
         ensure_in_epoch result result
 
-    ## Convert this time to text using the default formatter.
-
-       > Example
-         Convert the current time to text.
-
-             from Standard.Base import Date_Time
-
-             example_to_text = Date_Time.now.to_text
-    to_text : Text
```

Comment:
Are we generally deprecating to_text in favor of Javascript objects?

Reply:
no - just no need for this built-in wrapper. The other Time types don't have it so aligned with them.